src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonSerializer.Helpers.cs

```diff
@@ -18,10 +18,7 @@ private static JsonTypeInfo GetTypeInfo(JsonSerializerOptions? options, Type run
             Debug.Assert(runtimeType != null);
 
             options ??= JsonSerializerOptions.Default;
-            if (!options.IsInitializedForReflectionSerializer)
-            {
-                options.InitializeForReflectionSerializer();
-            }
+            JsonSerializerOptions.EnsureInitializedForReflectionSerializer();
```

Comment:
I'm guessing the original separation was to avoid making a method call if already initialized -- likely this was overcompensating but wondering if this has any noticeable impact on the benchmark suite.